src/layouts/components/Editor/extensions/Image/Image.ts

```diff
@@ -80,6 +90,40 @@ export const IsomerImage = Image.extend({
         editor.state.selection.replace(tr)
         return true
       },
+
+      setImageStyle: (styleOptions) => ({ tr, dispatch, editor }) => {
+        const { from, to } = tr.selection
+
+        let style = ""
+
+        if (styleOptions.width) {
+          if (styleOptions.width < 1 || styleOptions.width > 100) {
+            console.error(
+              "Invalid width value. It should be between 1 and 100."
+            )
+            return false
+          }
+          style += `width: ${styleOptions.width}%;`
+        }
+
+        if (styleOptions.backgroundColor) {
+          style += `background-color: ${styleOptions.backgroundColor};`
+        }
```

Comment:
can consider removing this first?

Reply:
Sure removed in 103897178315870ae1628dd281b0ef10b6d7f671